src/components/dictionaryConcepts/components/CreateConceptForm.jsx

```diff
@@ -69,10 +71,27 @@ const CreateConceptForm = (props) => {
               id="toggleUUID"
               onClick={props.toggleUUID}
             >
-            click here to manually enter
+                click here to manually enter
             </button>
           </div>
         </div>
+        )}
+        {(!isEditConcept) && (
+        <div className="form-row">
+          <div className="form-group col-md-7">
+            <label htmlFor="uuid"><h5>OCL ID</h5></label>
```

Comment:
This for attribute doesn't seem to be pointing to the right element.

Reply:
worked on this